dataset/src/test/kotlin/com/cosmotech/dataset/service/DatasetServiceImplTests.kt

```diff
@@ -59,6 +60,7 @@
         name = "My Dataset",
         description = "My Dataset description",
         organizationId = ORGANIZATION_ID,
+      security = DatasetSecurity(default = ROLE_NONE, accessControlList = mutableListOf(DatasetAccessControl(USER_ID, ROLE_NONE)))
```

Comment:
need to break the line because it gets over the default 120 char limit: e.g. 
```
        security = DatasetSecurity(default = ROLE_NONE,
                                   accessControlList = mutableListOf(DatasetAccessControl(USER_ID, ROLE_NONE)))
```

Reply:
fixed